src/styles/blocks/main_.scss

```diff
@@ -0,0 +1,12 @@
+.main {
+  &__wrapper {
+    margin-bottom: 50px;
+  }
+
+  &__cards {
+    display: flex;
+    justify-content: space-between;
+    flex-wrap: wrap;

```

Comment:
move it to `cards` selector. In BEM mix, element should have only external styles (margin, positioning)